lib/attachProof.js

```diff
@@ -120,13 +122,22 @@ async function attachAcceleratorProof(operation, {
  *
  * @returns {Promise<object>}
  */
-async function attachInvocationProof(operation, {
-  capability, capabilityAction, key, signer
+async function attachInvocationProof({
+  operation,
+  capability,
+  capabilityAction = 'write',
```

Comment:
is it dangerous to have 'write' as the default? Perhaps we shouldn't have a default here? Then again, there is no other useful choice for 'attach', is there?

Reply:
There is no other capability action for veres one, I'd say remove the param.